gulpfile.ts

```diff
@@ -113,3 +113,33 @@ gulp.task('test', (done: any) =>
   runSequence('build.test',
               'karma.start',
               done));
+
+// try it
+import * as express from 'express';
```

Comment:
Would you move this task definition in `/tools/tasks/seed/e2e.ts`? You'll get the exact same result.